src/options/Constants/PageConstants.cs

```diff
@@ -19,5 +19,6 @@ public static class PageConstants
         public const string PackageVersion = "Version Number";
         public const string OpenLinksInVS = "Open Links in VS";
         public const string EnableStartPagePlusOptions = "Start Page+ Options";
+        public const string AutoCloseOnSolutionOpen = "Automatically close Start Page+ when solution opens, show when solution closes";
```

Comment:
Could you change this to:

public const string EnableAutoCloseAndOpen = "Automatically close Start Page+ when solution opens, and open when solution closes";